dbcon/mysql/ha_view.cpp

```diff
@@ -108,8 +115,8 @@ void View::transform()
             else if (table_ptr->view)
             {
                 // for nested view, the view name is vout.vin... format
-                CalpontSystemCatalog::TableAliasName tn = make_aliasview(table_ptr->db.str, table_ptr->table_name.str, table_ptr->alias.str, viewName);
-                gwi.viewName = make_aliastable(table_ptr->db.str, table_ptr->table_name.str, viewName);
+                CalpontSystemCatalog::TableAliasName tn = make_aliasview(table_ptr->db.str, table_ptr->table_name.str, table_ptr->alias.str, viewName, lower_case_table_names);
```

Comment:
true before `lower_case_table_names` argument.

Reply:
Most excellent spot. I can't believe I missed one of those. I did triple check for just this.